helm/hpcc/values.schema.json

```diff
@@ -140,6 +140,44 @@
         ]
       }
     },
+    "dfuserver": {
+      "description": "dfuserver process",
+      "type": "array",
+      "items": {
+        "allOf": [
+          {
+            "name": {
+              "type": "string",
+              "description": "The name of the dfuserver process"
+            },
+            "disabled": {
+              "type": "boolean"
+            },
+            "enableSNMP": {
+              "type": "boolean",
+              "description": ""
+            },
+            "enableSysLog": {
+              "description": "",
+              "type": "boolean"
+            },
+            "queue": {
```

Comment:
I don't think we want to define a queue name separately in container mode.
I think want it to be implicit, just as it is for hthor, roxie, thor etc.
i.e. the queue is based on the name of the dfuserver